src/firmware_new/tools/debug_firmware_on_arm_board.sh

```diff
@@ -31,7 +31,8 @@ set +e
 # --- end runfiles.bash initialization ---
 
 # The directory this script is in
-CURR_DIR=$(dirname -- "$(readlink -f -- "$BASH_SOURCE")")
+CURR_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
+echo $CURR_DIR
```

Comment:
Is this just debug stuff? Also why not just update the `$BASH_SOURCE` instead of putting the command inline?

Reply:
hm this change wasn't supposed to make it, but `readlink` works differently on macOS and I was fiddling around with more agnostic methods